docker-32/install_miniconda.sh

```diff
@@ -0,0 +1,14 @@
+fname=$(ls Miniconda3-*-x86.sh | tail -n 1)
+# this is Ray Donnelly's custom Miniconda, built with our new gcc toolchain
+bash -x $fname -bfp /opt/miniconda
+rm $fname
+/opt/miniconda/bin/conda config --set show_channel_urls True
+# update --all is pulling in replacements from defaults
+# /opt/miniconda/bin/conda update --yes --all
+/opt/miniconda/bin/conda config --add channels c3i_test
+/opt/miniconda/bin/conda install --yes git conda-build curl anaconda-client
+/opt/miniconda/bin/conda install --yes conda-concourse-ci
```

Comment:
For now, until we get the 32-bit package built, this line can probably be

    # conda-install c3i dependencies, then pip-install c3i
    /opt/conda/bin/conda install -y contextlib2 networkx setuptools six yaml
    /opt/conda/bin/pip install https://github.com/conda/conda-concourse-ci/archive/master.zip